fe/fe-core/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -688,36 +693,36 @@ public void analyzeImpl(Analyzer analyzer) throws AnalysisException {
                 fn = getTableFunction(fnName.getFunction(), childTypes,
                         Function.CompareMode.IS_NONSTRICT_SUPERTYPE_OF);
                 if (fn == null) {
-                    throw new AnalysisException("Doris only support `explode_split(varchar, varchar)` table function");
+                    throw new AnalysisException(UNKNOWN_TABLE_FUNCTION_MSG);
                 }
-                return;
-            }
-            // now first find function in built-in functions
-            if (Strings.isNullOrEmpty(fnName.getDb())) {
-                Type[] childTypes = collectChildReturnTypes();
-                fn = getBuiltinFunction(analyzer, fnName.getFunction(), childTypes,
-                        Function.CompareMode.IS_NONSTRICT_SUPERTYPE_OF);
-            }
-
-            // find user defined functions
-            if (fn == null) {
-                if (!analyzer.isUDFAllowed()) {
-                    throw new AnalysisException(
-                            "Does not support non-builtin functions, or function does not exist: " + this.toSqlImpl());
+            } else {
```

Comment:
No need to change here

Reply:
We need to let the logic continue, so that we can cast the params of the function.